File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Data/Domain/PosteOnLine/DTO/POLRequestExtendedProperties/PRETNoticePushResponse.vb
Imports Newtonsoft.Json
''' <summary>
''' PRE = Pol Request Extended Properties
''' </summary>
Public NotInheritable Class PRETNoticePushResponse
    <JsonProperty("ReturnCode")>
    Public Property ReturnCode As TNoticeReturnCodeType

    <JsonProperty("Message")>
    Public Property Message As String

    <JsonProperty("TimeStamp")>
    Public Property TimeStamp As DateTime

    <JsonProperty("Reference")>
    Public Property Reference As String

    <JsonProperty("FatalError")>
    Public Property FatalError As TNoticeFatalErrorType
End Class

File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/PosteWeb/ExtendedProperties/PRETNoticePushResponse.ts
import TNoticeReturnCodeType = require("./TNoticeReturnCodeType");
import TNoticeFatalErrorType = require("./TNoticeFatalErrorType");

interface PRETNoticePushResponse {
    ReturnCode: TNoticeReturnCodeType,
    Message: string,
    Timestamp: Date,
    Reference: string,
    FatalError : TNoticeFatalErrorType
}

export = PRETNoticePushResponse